edx_exams/apps/api/serializers.py

```diff
@@ -94,3 +94,14 @@ class Meta:
             "id", "created", "modified", "user", "start_time", "end_time",
             "status", "exam", "allowed_time_limit_mins", "attempt_number"
         )
+        
+        """ 
+        
+        # Serializer fields from edx-proctoring:
```

Comment:
Is this a note to yourself, or does this have a purpose? If it's just a helpful note to yourself, don't forget to remove it!